### PR TITLE
CORE-6561 remove method from signing service that is not needed 

### PR DESCRIPTION
**Pull request by LWogan**
Remove method from signing service to decode public keys which was used in tests. Public keys can now be retrieved from memberInfo via the member lookup service

### CODE REVIEW

**Review by lankydan — COMMENTED** [review]
Would it be better to put this into dp2 if its not too late?

**Comment by JamesHR3** [comment]
> Would it be better to put this into dp2 if its not too late?

I'd prefer not to put things into DP2 at this point unless it's a bug fix. It's a bit unfortunate having this present but I don't think it's the end of the world.

File: components/flow/flow-service/src/main/kotlin/net/corda/flow/application/services/MemberLookupImpl.kt [hunk 1]
```diff
@@ -11,32 +13,37 @@ import org.osgi.service.component.annotations.Activate
...
-import java.security.PublicKey
 
 @Suppress("Unused")
 @Component(service = [MemberLookup::class, SingletonSerializeAsToken::class], scope = ServiceScope.PROTOTYPE)
 class MemberLookupImpl @Activate constructor(
     @Reference(service = FlowFiberService::class)
     private val flowFiberService: FlowFiberService,
 ) : MemberLookup, SingletonSerializeAsToken {
+
+    @Suspendable
```
Comment (JamesHR3) [inline components/flow/flow-service/src/main/kotlin/net/corda/flow/application/services/MemberLookupImpl.kt:24]: Were these causing problems? I'm a bit worried that implies we have a fairly serious bug in DP2.

**Review by JamesHR3 — COMMENTED** [review]
Looks reasonable but needs the corresponding API PR updating.

**Review by mnesbit — APPROVED** [review]
OK

**Review by ronanbrowne — COMMENTED** [review]
lgtm (build)

**Review by ronanbrowne — APPROVED** [review]

**Pull request merged**